ansible/roles/nginx/templates/aaad.conf.upstart.j2

```diff
@@ -6,7 +6,7 @@ stop on runlevel [0156]
 chdir {{nginx_conf_dir}}/aaad
 
 env HTPASSWD_FILE={{ nginx_conf_dir }}/{{ nginx_basic_auth_file_name }}
-env PERMISSIONS_FILES={{ nginx_conf_dir }}/permissions.yml
+env PERMISSIONS_FILES={{ nginx_conf_dir }}/permissions.yml{% if set_test_users is defined and set_test_users|bool %},{{ nginx_conf_dir }}/permissions_test_users.yml{% endif %}
```

Comment:
`set_test_users` as a var, is it shared? If it is specific to the nginx role, we probably want to prefix it with `nginx_` to prevent it from colliding.